schemas/cloud-gateway-schema.json

```diff
@@ -32,6 +32,80 @@
                                                 }
                                             }
                                         },
+                                        "security" :{
+                                            "type": "object",
+                                            "description": "Security settings",
+                                            "properties": {
+                                                "oauth2": {
+                                                    "type": "object",
+                                                    "description": "Enable OAuth2 login flow",
+                                                    "properties": {
+                                                        "client": {
+                                                            "type": "object",
+                                                            "description": "OAuth2 client settings",
+                                                            "properties": {
+                                                                "registration": {
+                                                                    "type": "object",
+                                                                    "description": "OAuth2 client registration settings",
+                                                                    "patternProperties": {
+                                                                        "$*.^": {
```

Comment:
identity provider name needs to go right after "client" and before "provider" and "registration". Here is an example of how code expects the format of the variable:
ZWE_configs_spring_security_oauth2_client_okta_provider_authorizationUri
ZWE_configs_spring_security_oauth2_client_okta_registration_clientId